app/models/review.rb

```diff
@@ -0,0 +1,15 @@
+class Review < ApplicationRecord
+  belongs_to :user
+  belongs_to :project
+
```

Comment:
I like these star methods! These might be really cool to move into a helper if they are specifically view related